contrib/mesos/pkg/executor/service/service.go

```diff
@@ -43,10 +46,16 @@ import (
 	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
 )
 
+// envContainerID is the name of the environment variable that contains the
+// Mesos-assigned container ID of the Executor.
```

Comment:
probably add a TODO about how this can be ripped out once we have mesos native containerization support factored into a custom kubelet runtime